Loop/Managers/LoopDataManager.swift

```diff
@@ -1130,6 +1130,7 @@ extension LoopDataManager {
     ///     - LoopError.missingDataError
     ///     - LoopError.configurationError
     ///     - LoopError.glucoseTooOld
+    ///     - LoopError.glucoseInFuture
```

Comment:
nit: maybe `invalidFutureGlucose`?  Not a blocker, but "glucose in future" doesn't seem to convey any kind of problem.

Reply:
I agree. Done.